content/response-to-reviews/therapeutics-v1.md

```diff
@@ -0,0 +1,87 @@
+**
+
+More importantly, the conclusion for all but 1-2 therapeutics is that more data is needed, leaving me wondering what the value of this review is at time present. One useful contribution would be to rank the promise of each approach. Each has issues, but which ones are worth investing more resources in?
+
+**
+We hope that the recontextualization of the content serves to address the concerns related to the usefulness of the project -- the goal is to organize the available evidence to allow scientists to get or stay up to speed on the status of these therapeutics without needing to dive independently into the exceptionally large body of literature.
+**
+
+1.4 Figure 1 - I'm not sure this adds much to the current topic.
+
+**
+Figure 1 has been modified to show the relative trajectories of the SARS-CoV-1 and SARS-CoV-2 viral outbreaks (data were not available for MERS).
+The goal of this figure is to show why pharmacology plays such a bigger role in the present pandemic than it has in the past, as well as to emphasize how much more feasible clinical trials are based on the differences in caseload.
+We hope the relevance is improved with these changes.
+**
+
+1.5 Figure 2 - panel labels (letters) are needed. 
+
+**
+This change has been made.
+**
+
+The y-axis of panel 4 is difficult to read - the panel should either be stretched or some of the less common approaches could be trimmed off. The "other" category could also be removed.
+
+**
+These changes have been made.
+**
+
+More importantly, I worry that this figure is immediately out of date, is there any indication that these trends will be stable over time? If not, why is it important to know what fraction of trials are in Phase 2 right now?
+
+**
+As part of this project, we developed software to pull data from the web and update these figures every night.
+While the version of record will be locked at a certain time point, this approach does allow us to keep the version online continually updated.
+The stability of these trends over time is an interesting question that it would be possible to address using these data sources.
+To date, we had thought of the utility of the figure as showing what research was in progress and therefore what evidence would be expected to eventually become available.
+**
+
+1.6 Figure 3 - I really liked this figure and would like to see it earlier in the review. One concern I have is that all of the drugs are equivalent on this diagram, which could lead the reader to think that HCQ/CQ are effective. Perhaps it would be safer to remove the drug names and indicate the various mechanisms that could be theoretically targeted. Or alternatively, the thickness of the arrows could be used to indicate current data on efficacy.
+
+**
+Thank you for your positive feedback and thoughtful critique of this figure.
```

Comment:
I think changing the color is a good idea to differentiate those still of interest or not.

Reply:
I agree. I can do that. So only green and blue to differentiate the two or are we confident in a more graded differentiation (like 3 colors for non-effective, inconclusive and effective)? Either way, it will only take a few minutes to change these things. Let me know if there is anything else that should be changed while we are at it.

Reply:
I would let @rando2 decide, but if you want to do the graded approach you could use David Fajgenbaum's graded system to cite and advise on that. https://cdcn.org/corona-data-viewer/

Reply:
This is amazing @RLordan! Yes, let's use that system -- the reviewer had suggested we should rank them according to promise, which I wasn't sure how to do, but this is a perfect solution!!!

@nilswellhausen I don't know mean to say you have to red and blue, those were just the ones that came to mind since the names are already red anyways -- definitely your aesthetic eye is better than mine! @vincerubinetti is also very good at visual design and is now a member of this project because he worked on some of the Manubot stuff -- tagging him in case he has any opinions!

Reply:
I can help if necessary, though I can't seem to find/figure out which diagram these comments are referring to in the appveyor preview. Perhaps someone could screenshot?

If you just need some nice colors, here is a palette I use a lot: https://www.materialpalette.com/colors

Reply:
Oops sorry @vincerubinetti! Here it is:
![N001-LifeCyclePlusDrugs](https://user-images.githubusercontent.com/8975841/127909264-bf9d2dfd-d584-413a-ac4d-597cd8e2ec8f.png)

Reply:
I might try coloring the effective drugs `#8e24aa`, the less effective `#ba68c8`, and the non-effective to `#e1bee7`. Just an idea, sounds like you have alternatives that will be fine too.

Reply:
Probably a good idea to color the drugs differently than the other red text at the top, which (despite my limited biology knowledge) I'm fairly sure are not drugs.

Reply:
@vincerubinetti Which other red text are you referring to? The only red text are drugs or drug classes in the case of Ace inhibitors and neutr. abs. I can follow David/s grading scheme (which is A,B,C,D) and convert it into a color legend? Maybe a dark green for A, lime/yellow for B, deep orange for C and red for D?

Reply:
Nevermind, I didn't think "neutralizing antibodies" was a drug class. Your plan sounds good to me.

Reply:
![COVID life cycle and drugs (3)](https://user-images.githubusercontent.com/62564517/127929319-35c0c957-6ed4-4ea9-ad59-fd73a8c0ae96.png)

Reply:
I changed the color according to the grading system. Figure legends should include legend (Green =A, Lime=B, Orange=C and Red = D

Reply:
This looks great! Thank you both! 
File replacement proposed over at #986